etc/setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# install
+mkdir ~/etc/
+FILE=~/etc/var.sh
+if [ ! -f $FILE ];
+then
+    >$FILE
+    echo "PythonFolder=$(dirname "$(which python)")" >> $FILE
```

Comment:
make sure check the python version and quite the program if the version is lower than the minimum working version.